### DISCUSSION TITLE
Idk wtf just happened

### DISCUSSION THREAD

**Discussion by Asvarius** [Vedza/NitroSniperGo#282] [General]
so, I decided to turn off the sniper for a couple days, today I sniped with it off? like wtf, I sniped like 4 codes in a row.

**Comment by Vedza** [comment]
Wtf, are you sure there isn't one running somewhere ?
If you're on linux or mac execute `ps -aux | grep Nitro`.

**Reply by Asvarius** [reply]
yeah, I found out what happened, It was running in background on my other pc
